internal/generator/suite.go

```diff
@@ -191,3 +193,68 @@ func (s *Suite) String() string {
 
 	return spaceRegex.ReplaceAllString(strings.TrimSpace(result.String()), "\n")
 }
+
+const bashSuiteTemplate = `
+function setup() {
+	{{ .Setup }}
+}
+
+function cleanup() {
+	{{ .Cleanup }}
+}
+`
+
+// BashString generates bash script for the suite
+func (s *Suite) BashString() string {
+	setup := strings.Join(s.getCompleteSetup(), "\n")
```

Comment:
We also need to add something like `|| exit` after each command, so that if retry fails, we exit the script.
Maybe something like this:
```bash
function retryCommand() {
while ! timeout && ! bash -c "$1"
do; : ; done
}
function setup() {
retryCommand "kubectl apply -k path" || exit
retryCommand "kubectl wait" || exit
retryCommand "kubectl exec ping" || exit
}
```